poseidon-hs.cabal

```diff
@@ -58,7 +58,8 @@ Test-Suite poseidon-tools-tests
                         Poseidon.SummariseSpec, Poseidon.SurveySpec, Poseidon.GenotypeDataSpec,
                         Poseidon.EntitiesListSpec, PoseidonGoldenTests.GoldenTestsValidateChecksumsSpec,
                         PoseidonGoldenTests.GoldenTestsRunCommands, Poseidon.ChronicleSpec,
-                        Poseidon.SequencingSourceSpec, Poseidon.InterfaceSpec
+                        Poseidon.SequencingSourceSpec, Poseidon.InterfaceSpec,
```

Comment:
I think `InterfaceSpec` was specifically meant as `OptParseApplicativeParsersSpec`. We only need one module to test code form `OptparseApplicativeParsers.hs`, right? I say: You choose which one you prefer and we remove the other one.

Reply:
True, thanks for pointing that out. I will move this over, or merge.